frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {
+  getContainerImageByDisk,
+  getURLSourceByDisk,
+  getPVCSourceByDisk,
+  getWindowsToolsURLByDisk,
+} from '../../selectors/vm/selectors';
+
+const getDiskSource = (vm, { name }) => {
+  const container = getContainerImageByDisk(vm, name);
```

Comment:
I would prefer to have a function similar to https://github.com/kubevirt/web-ui-components/blob/master/src/selectors/dv/selectors.js#L17 which would replace `getContainerImageByDisk`, `getPVCSourceByDisk` and `getURLSourceByDisk`

Reply:
I wish I knew about this function! btw i don't see a way to determine the container image by this function, am i missing something?

Reply:
there was no need. Feel free to add it :)